### PR TITLE
Replace blog pagination with generic component

### LINKED ISSUE #6286: /blog - fix middot on the pagination
I think this will be solved if we update to the latest Vanilla.

Screenshot 2019-12-17 at 10 54 31

---
*Reported from: https://ubuntu.com/blog*

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Replaced blog pagination with a generic pagination component which can be reused, specifically for `/tutorials`.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://ubuntu-com-canonical-web-and-design-pr-6343.run.demo.haus/blog
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that the pagination works
- Check that the pagination matches the [vanilla pattern](https://docs.vanillaframework.io/patterns/pagination/)

## Issue / Card
Fixes #6286

## Screenshots
blog_pagination_screenshot

### CODE REVIEW

**Comment by tbille** [comment]
https://ubuntu-com-canonical-web-and-design-pr-6343.run.demo.haus/blog/desktop

not working.

File: static/sass/styles.scss [hunk 1]
```diff
@@ -479,3 +479,9 @@ summary {
     justify-content: flex-start;
   }
 }
+
+// XXX - vanilla issue to add this variant https://github.com/canonical-web-and-design/vanilla-framework/issues/2729
+.p-pagination--centered {
```
Comment (sowasred2012) [inline static/sass/styles.scss]: You could also achieve this by adding `.u-align--center` on the `ol` element itself, removing the need for this modifier class - my instinct initially was to add that class to the `ol`s parent element, which makes me wonder if this suggestion is the best approach.

Comment (steverydz) [reply static/sass/styles.scss]: thanks - I've updated it

**Comment by steverydz** [comment]
@tbille @sowasred2012 fixed

**Pull request merged**